src/sentry/ingest/inbound_filters.py

```diff
@@ -253,8 +253,8 @@ class _LegacyBrowserFilterSerializer(_FilterSerializer):
 
 _healthcheck_filter = _FilterSpec(
     id=FilterStatKeys.HEALTH_CHECK,
-    name="Filter out requests to health check endpoints",
-    description="Filters out transactions for health check endpoints, in order to conserve transaction quotas",
+    name="Filter out health check transactions",
+    description="Filter transactions that match most common naming patterns for health checks.",
```

Comment:
Do we have a user-visible list of what these patterns are? Do we want to expose that list to users?

Reply:
They are the same as [here in docs](https://docs.sentry.io/product/performance/performance-at-scale/?original_referrer=https%3A%2F%2Fwww.google.com%2F#health-checks.). Radu is working on docs for this inbound filter as we speak. Ideally, we would link there from this message, but it's a message generated by the backend so we can't inject HTML link. Perhaps in a follow-up.